Sources/Plot/API/HTMLComponents.swift

```diff
@@ -579,6 +579,96 @@ extension List: ComponentContainer where Items == ComponentGroup {
     }
 }
 
+// Component used to render an `<picture>` element for responsive image support.
+public struct Picture: Component {
+    /// Type used to define an  Picture source, which points to an image file (or set of them) and associated media queries (and other paramaters).
```

Comment:
Also a typo here: Should be "define a `Picture` source"

Reply:
As above, so bellow.